### PR TITLE
[Issues 1259] add frontend make build target

### LINKED ISSUE #1259: [Task]: Add new make target to build nextjs service
### Summary

When adding new frontend packages, you may need to rebuild locally for the new packages to be copied into the container

### Acceptance criteria

- new make build target added

### PR DESCRIPTION
**Pull request by rylew1**
## Summary
Fixes #1259 

## Changes proposed
- add frontend make build target

## Context for reviewers
- when adding a new package with npm and running `make dev`, local docker needs to rebuild to pull in new dependencies

### CODE REVIEW

File: frontend/Makefile [hunk 1]
```diff
@@ -40,6 +40,9 @@ release-build:
 ##################################################
 # Local development
 ##################################################
+build: # Build the Next.js local dev server in Docker
+	docker-compose build --no-cache nextjs
+
```
Comment (rylew1) [inline frontend/Makefile:45]: There might be a better option for updating local packages in docker - but this was the quick solution

File: frontend/Makefile [hunk 2]
```diff
@@ -40,6 +40,9 @@ release-build:
 ##################################################
 # Local development
 ##################################################
+build: # Build the Next.js local dev server in Docker
+	docker-compose build --no-cache nextjs
```
Comment (SammySteiner) [inline frontend/Makefile:44]:
```suggestion
	docker compose build --no-cache nextjs storybook
```
docker compose doesn't get a dash anymore, I also type it accidentally all the time.
let's add storybook here too, unless you want to have separate build-dev and build-storybook make commands.
can you also update our renovate review instructions with this make command [here](https://app.gitbook.com/o/cFcvhi6d0nlLyH2VzVgn/s/v1V0jIH7mb7Yb3jlNrgk/engineering/draft-renovate-pr-playbook-and-workflow/~/comments?context=post&node=d827e994cee6430dbbbdcc257800ff64#:~:text=%3CCtrl%3E%20c-,run%20docker%20compose%20build%20%2D%2Dno%2Dcache%20nextjs%20storybook%20to%20recreate%20all%20layers%20of%20your%20docker%20build%20image%20for%20the%20frontend%20app%20and%20storybook,-run%20make%20%2DC)

**Review by SammySteiner — APPROVED** [review]
Approving, but please remove the dash from `docker-compose` before merging as that [has been deprecated](https://docs.docker.com/compose/reference/)

File: frontend/Makefile (hunk 2 above)
Comment (rylew1) [reply frontend/Makefile:44]: I propose splitting splitting up the app and storybook to make it more modular

**Pull request merged**